### ISSUE TITLE
Consent Mode disconnection modal: Future proof copy and/or link to EU user consent policy.

### ISSUE THREAD

**Issue by techanvil** [google/site-kit-wp#8655]
## Feature Description

As [discussed](https://github.com/google/site-kit-wp/issues/8643#issuecomment-2086010916) on #8643, it could be worth amending the copy on the Consent Mode disconnection confirmation modal to be more future proof in case of any further modifications to the list of countries Google's EU user consent policy applies to.

As per #8643, at the time of writing the copy is as follows:

_Modal when Ads conversion tracking is detected:_
![image](https://github.com/google/site-kit-wp/assets/18395600/bde0b061-3e23-4e5b-b968-3439dc2b091d)

_Modal when Ads conversion tracking is not detected:_
![image](https://github.com/google/site-kit-wp/assets/18395600/2accf9df-01e1-4c0a-bb0f-6492e09c8662)

However once #8643 has been implemented the copy is expected to be respectively:

* _Disabling consent mode may affect your ability to track these in the European Economic Area, the UK and Switzerland:_
* _Disabling consent mode may affect your ability in the European Economic Area, the UK and Switzerland:_

We could consider referring to "EU user consent policy regions" instead of the explicit set of regions, or something along those lines.

If so, or possibly even if not, it could be worth linking to the policy from the modal to help understand the wider context.

It's worth noting the EU user consent policy _is_ linked to from the [SK support document](https://sitekit.withgoogle.com/documentation/using-site-kit/consent-mode/) that's linked to from the "Learn more", and "See suggested plugin" links - maybe we could link here instead of directly to the policy if adding a link to the modal.

![image](https://github.com/google/site-kit-wp/assets/18395600/335b051c-34eb-4537-9e0f-9a8d6c473e7f)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* [ ] 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by benbowler** [comment]
Update June 2024:

#8643 has now been implemented, awaiting review and release after July 31st. The copy in the ticket lists all regions rather than making reference to "EU user consent policy regions". This ticket therefore, is suggesting we add a "Learn more" link to the "Disable consent mode" modal linking to the policy docs.

Personally I don't think this adds much value as the policies are well linked on the consent mode screen which you have to visit to trigger this dismiss modal. I think we communicate to the user the effect of their action without adding an additional link here.

**Comment by techanvil** [comment]
Thanks @benbowler - as mentioned on [Slack](https://10up.slack.com/archives/CBKKQEBR9/p1718276125979869?thread_ts=1717579271.410249&cid=CBKKQEBR9), let's get @aaemnnosttv's thoughts on how to proceed here.

**Comment by benbowler** [comment]
Hey @techanvil, I got some time with @aaemnnosttv on our 1:1 today to discuss there. We talked around it for a little while and we landed here: Updating the copy would save us from changing this component in future however, we will need to update the docs and the list of configured regions whenever regions change in the future anyway and there's low overhead to update these strings along with that work. There are plenty of learn more links in the Consent Mode settings, so adding a fourth in the modal is not a clear benefit.

The decision was, let's close this issue and revisit it if region changes become common and this optimisation makes sense.
